src/content/configuration/other-options.md

```diff
@@ -297,6 +298,72 @@ module.exports = {
 
 W> `cache.idleTimeoutForInitialStore` is only available when [`cache.store`](#cachestore) is set to `'pack'`
 
+### cache.maxMemoryGenerations
+
+`number`
```

Comment:
Since those are new features introduced in webpack 5.30.0, let's use `<Badge text='5.30.0+' />` to indicate the version when those features are available.